src/components/AddressCard.tsx

```diff
@@ -124,8 +124,9 @@ const AddressCard = ({ style, addressHash, onSettingsPress }: AddressCardProps)
           shadowOffset: { height: 5, width: 0 },
           shadowOpacity: theme.name === 'dark' ? 0.5 : 0.15,
           shadowRadius: 8,
-          elevation: 10,
-          borderColor: outterBorderColor
+          elevation: 3,
+          borderColor: outterBorderColor,
+          overflow: 'hidden'
```

Comment:
I had to change the elevation and also add the overflow, cause it didn't look as good on Android as it did on iOS:

<img width="489" alt="image" src="https://github.com/alephium/mobile-wallet/assets/1579899/0afcc98d-1294-45f1-b439-5bf9de503a93">